components/camel-chatscript/src/main/docs/camel-chatscript-component.adoc

```diff
@@ -0,0 +1,77 @@
+[[ChatScript-component]]
```

Comment:
adoc file starting with camel- is not correct, please remove one of doc files

Reply:
done